src/browser/reducers/requestsReducer.js

```diff
@@ -42,6 +43,10 @@ export default function requestsReducer(state = initialState, action) {
       return dequeue(state, action.sessionId, action.id);
     case EMPTY_REQUESTS:
       return empty(state, action.sessionId);
+    case EMPTY_ALL_REQUESTS:
+      return initialState;
+    case CLOSE_TAB:
```

Comment:
This can be combined with the `EMPTY_REQUESTS` case, i.e.:

```js
case EMPTY_REQUESTS:
case CLOSE_TAB:
  return empty(state, action.sessionId);
```

Reply:
if that's how the codebase is 👍